common/sshkeys/sshkeys.go

```diff
@@ -107,65 +107,56 @@ func loadSSHKeysOfCurrentUser() {
 
 	// Ask whether to load private keys or provide own key
 	options := []string{
-		"Load private ssh keys from " + privateKeyDir,
+		fmt.Sprintf("Load the private SSH keys from the directory '%s'" + privateKeyDir),
```

Comment:
Where is the privateKeyDir available? Is it provided as input or do we pick up `.ssh` dir on the client machine?

Reply:
Does this function read the `.ssh` directory of the user?